app/models/address.rb

```diff
@@ -0,0 +1,3 @@
+class Address < ApplicationRecord
+  belongs_to :user
```

Comment:
No validation of fields?
Will address always belongs to a user?
Can't be like billing address, shipping address etc

Reply:
Need to discuss this for  ```Address Type```